apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -220,7 +220,7 @@ const EditorialSwitchSectionCodec = t.strict({
   __component: t.literal('sections.editorial-switch'),
   theme: t.union([t.literal('light'), t.literal('dark')]),
   title: t.string,
-  subtitle: t.union([t.string, t.null]),
+  body: t.union([t.string, t.null]),
```

Comment:
The issue is that the field is not a Rich Text and as such is not rendered as markdown.

Changing the field's name does not solve the issue. Turn `body` back to `subtitle` everywhere for consistency.